CGPs/template.md

```diff
@@ -0,0 +1,41 @@
+# CIP [0000]: Title for Governance Proposal
+
+- Date: 2020-05-11
+- Author: @githubhandle
+- Status: DRAFT|PROPOSED|EXECUTED|EXPIRED
+	- DRAFT = Feedback collection
+	- PROPOSED = The governance proposal has been submitted to the network
```

Comment:
"is" or "was" ("has been" generally less  readable)

Reply:
+1 to "was"